hil/api.py

```diff
@@ -1233,8 +1233,7 @@ def list_headnode_images():
 def list_active_extensions():
     """List all active extensions"""
     get_auth_backend().require_admin()
-    extensions_raw = cfg.items('extensions')
-    extensions = sorted([ext[0] for ext in extensions_raw])
+    extensions = [ext[0] for ext in cfg.items('extensions')]
```

Comment:
You can keep the call to `sorted`; sorry, my comment had been on the commit before you added that.